lewis/scripts/control.py

```diff
@@ -41,16 +41,20 @@ def show_api(remote, object_name):
     methods = []
 
     for member_name in dir(obj):
-        if not member_name[0] in ('_', ':'):
-            member = getattr(obj, member_name)
-
-            if callable(member):
-                methods.append(member_name)
+        if member_name[0] not in ('_', ':') and member_name not in dir(type(obj)):
+            methods.append(member_name)
```

Comment:
Actually, to be a bit more precise, this is where the proxy objects are not always 100% accurate representations of the server side objects. I guess it's a trade-off between ease of implementation and difficulty to tell that it's a proxy object. At least at the moment I think this is acceptable, because you have to go and get the object explicitly anyway, so it's pretty clear that it's a server side object. The JSON serialization poses way more constraints anyway (but again, for the scope this is intended I find that acceptable).